File: DSA/Vyom-Singhal/Day3/Day3-q1.cpp
#include<iostream>
using namespace std;

bool isPangram(string str) {
    bool alphabetPresent[26] = {false};
    for(int i = 0; i < str.length(); i++) {
        char alpha = tolower(str[i]);
        if(alpha >= 'a' && alpha <= 'z') alphabetPresent[alpha-'a'] = true;
    }
    for(int i = 0; i < 26; i++) {
        if(alphabetPresent[i] == false) return false;
    }
    return true;
}

int main() {
    string sentence;
    cout << "Enter sentence : ";
    getline(cin,sentence);
    bool ans = isPangram(sentence);
    if(ans == true) cout << "complete";
    else cout << "incomplete";
}

File: DSA/Aanidhay-Aggarwal/Day-3/Day3_q1.java
import java.util.Scanner;

public class PangramChecker {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter a sentence: ");
        String sentence = scanner.nextLine();

        boolean isPangram = true;
        for (char c = 'a'; c <= 'z'; ++c) {
            if (!sentence.toLowerCase().contains(String.valueOf(c))) {
                isPangram = false;
                break;
            }
        }

        System.out.println(isPangram ? "complete" : "incomplete");
    }
}
